UnitTests/GitCommands.Tests/Git/ObjectIdTests.cs

```diff
@@ -291,19 +291,13 @@ public void ToShortString()
         }
 
         [Test]
-        public void Equals_with_string()
+        public void Equals_with_random()
         {
             for (var i = 0; i < 100; i++)
             {
                 var objectId = ObjectId.Random();
-                Assert.True(objectId.Equals(objectId.ToString()));
+                Assert.True(objectId.Equals(objectId));
             }
-
-            Assert.False(ObjectId.Random().Equals((string)null));
-            Assert.False(ObjectId.Random().Equals(""));
-            Assert.False(ObjectId.Random().Equals("gibberish"));
-            Assert.False(ObjectId.Parse("0123456789012345678901234567890123456789").Equals(" 0123456789012345678901234567890123456789 "));
-            Assert.False(ObjectId.Parse("aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa").Equals("AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"));
         }
 
```

Comment:
This test has become obsolete by removing the overload. There are other testcases for `.Equals(ObjectId)`.

Reply:
I only kept it as a kind of stress test, but will update other or just remove.

Reply:
removed the test

plan to merge together with the related commits, keeping them together